source/installing-splunk/splunk-distributed.rst

```diff
@@ -70,50 +73,204 @@ Each instance can be installed on different hosts following the same steps descr
+  - **inputs.conf**: Reads alerts from **alerts.json**
+  - **outputs.conf**: Contains necessary information to know the cluster's indexers.
+  - **props.conf**: Contains the different ways of how alerts will be indexes.
 
-  You must run this command for each search peer that you want to add.
+Starting with **inputs.conf**, create it and fill it with the next block:
 
-  **Note the following:**
+.. code-block:: console
 
-  1. ``<host>`` is the host name or IP address of the search peer's host machine.
-  2. ``<port>`` is the management port of the search peer. By default it's 8089.
-  3. The ``-auth`` flag is used to provide credentials for the search head.
-  4. The ``-remoteUsername`` and ``remotePassword`` flags are used to provide the credentials for the search peer. The remote credentials must be for an admin-level user on the search peer.
+  # touch /opt/splunkforwarder/etc/system/local/inputs.conf
 
-  .. warning::
-    If there are login issues when trying to add the search peer, add the ``allowRemoteLogin = always`` option under the ``[general]`` section on the ``/opt/splunk/etc/system/local/server.conf`` file, and then restart the search peer.
+.. code-block:: xml
 
-2. On the **search peer** instance we need to add the files to configure the Wazuh indexes:
+  [monitor:///var/ossec/logs/alerts/alerts.json]
+  disabled = 0
+  host = MANAGER_HOSTNAME
+  index = wazuh
+  sourcetype = wazuh
 
-  a) Download and insert the ``inputs.conf`` template to configure where the data will come from:
+  Now, following with the **outputs.conf**:
 
-    .. code-block:: console
+.. code-block:: console
+
+  # touch /opt/splunkforwarder/etc/system/local/outputs.conf
+
+And paste this inside:
+
+.. code-block:: xml
+
+  [indexer_discovery:cluster1]
+  pass4SymmKey = changeme
+  master_uri = https://<master_ip>:<port>
+
+  [tcpout:cluster1_tcp]
+  indexerDiscovery = cluster1
+
+  [tcpout]
+  defaultGroup = cluster1_tcp
+
+For the last one, the **props.conf**, follow the same procedure:
+
+.. code-block:: console
+
+  # touch /opt/splunkforwarder/etc/system/local/props.conf
+
+.. code-block:: xml
+
+  [wazuh]
+  DATETIME_CONFIG =
+  INDEXED_EXTRACTIONS = json
+  KV_MODE = none
+  NO_BINARY_CHECK = true
+  category = Application
+  disabled = false
+  pulldown_type = true
+
+To save all the changes, restart splunk:
+
+.. code-block:: console
+
+  # /opt/splunkforwarder/bin/splunk restart
+
+
+Indexers:
+_________
+
+Here, in the **master instance**, make the configuration that will be pushed to the indexers.
+
+For this configuration is necessary to create the following two files and paste into them the following blocks of code respectively:
+
+.. code-block:: console
+
+  # touch /opt/splunk/etc/master-apps/_cluster/local/inputs.conf
+
+.. code-block:: xml
+
+  [splunktcp://9997]
+  connection_host = ip
+
+
+.. code-block:: console
+
+  # touch /opt/splunk/etc/master-apps/_cluster/local/indexes.conf
```

Comment:
This process can be fully replaced by this command:
```
curl -so /opt/splunk/etc/system/local/indexes.conf https://raw.githubusercontent.com/wazuh/wazuh/3.7/extensions/splunk/peer-indexes.conf
```
So, please remove the `touch` command and just show the content of the `indexes.conf` file in order to users can understand the content of what they're installing on their instances.